client/src/components/ImageGallery.jsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { Grid, Col, Row } from 'react-styled-flexboxgrid';
```

Comment:
For all of these, did you have to convert everything to styled components from flexboxgrid? Or was this unused originally? Just curious

Reply:
It was unused originally.